srv/salt/ceph/updates/default-zypper-kernel-default.sls

```diff
@@ -0,0 +1,22 @@
+
+
+zypper update:
+  cmd.run:
+    - name: "zypper --non-interactive  update --replacefiles --auto-agree-with-licenses"
+    - shell: /bin/bash
+    - unless: "zypper lu | grep -sq 'No updates found'"
+
+kernel install:
+  cmd.run:
+    - name: "zypper --non-interactive --no-gpg-checks install --force-resolution kernel-default"
+    - shell: /bin/bash
+    - unless: "rpm -q kernel-default"
+    - fire_event: True
+
+kernel update:
```

Comment:
why do you need this rule/step to update the kernel? the above `zypper update` already updates all installed packages even the kernel packages